cli/daemon/dash/dashapp/src/main.tsx

```diff
@@ -7,7 +7,7 @@ import { createRoot } from "react-dom/client";
 const container = document.getElementById("root");
 const root = createRoot(container!);
 root.render(
-  <React.StrictMode>
-    <App />
-  </React.StrictMode>
+  // <React.StrictMode>
```

Comment:
Disabling strict mode for now. Have a ticket to go through the code and fix some issues so that we can enable this again.